content/guides/core/reports/edit-a-report.md

```diff
@@ -82,7 +82,18 @@ Add run sets from projects interactively with the App UI or the W&B SDK.
 {{< tabpane text=true >}}
 {{% tab header="App UI" value="app" %}}
 
-Enter a forward slash (`/`) in the report to display a dropdown menu. From the dropdown, choose Panel Grid. This will automatically import the run set from the project the report was created from.
+Enter a forward slash (`/`) in the report to display a dropdown menu. From the dropdown, choose **Panel Grid**. This will automatically import the run set from the project the report was created from.
+
+If you import a panel into a report, run names are inherited from the project. In the report, you can optionally [rename a run]({{< relref "/guides/models/track/runs/#rename-a-run" >}}) to give the reader more context. The run is renamed only in the individual panel. If you clone the panel in the same report, the run is also renamed in the cloned panel.
+
+1. In the report, click the pencil icon to open the report editor.
+1. In the run set, find the run to rename. Hover over the report name, click the three vertical dots, then select either:
+
+  - **Rename run for project**: rename the run across the entire project. To generate a new random name, leave the field blank.
+  - **Rename run for panel grid** rename the run only in the report, preserving the existing name in other contexts. Generating a new random name is not supported.
+
+  Submit the form.
```

Comment:
Missing number that continues numbered list. Also, extra empty line in between list items.

Reply:
Needed an extra level of indent.

In Jekyll, omitting a newline before starting a list causes things to break. It doesn't seem to break things here and in this case makes it obvious that Submit the form is part of the prior step, I think.